### PR TITLE
refactor: exclude mls-related conference logic to separate module

### PR DESCRIPTION
**Pull request by PatrykBuniX**

### CODE REVIEW

File: src/script/calling/mlsConference.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ *
+ */
+
+import {Subconversation} from '@wireapp/api-client/lib/conversation/Subconversation';
+import {QualifiedId} from '@wireapp/api-client/lib/user';
+import {MLSService} from '@wireapp/core/lib/messagingProtocols/mls';
+
+import {SubconversationEpochInfoMember} from './CallingRepository';
+
+export const generateSubconversationMembers = async (
```
Comment (atomrc) [inline src/script/calling/mlsConference.ts]: I believe this one can stay private (no need to export it)

Comment (PatrykBuniX) [reply src/script/calling/mlsConference.ts]: correct

File: src/script/view_model/CallingViewModel.ts [hunk 2]
```diff
@@ -304,7 +225,11 @@ export class CallingViewModel {
 
       const subconversation = await mlsService.getConferenceSubconversation(conversationId);
 
-      const {epoch, keyLength, secretKey, members} = await getSubconversationEpochInfo(conversationId, subconversation);
+      const {epoch, keyLength, secretKey, members} = await getSubconversationEpochInfo(
+        {mlsService},
+        conversationId,
+        subconversation,
+      );
       this.callingRepository.setEpochInfo(conversationId, {epoch, keyLength, secretKey}, members);
```
Comment (atomrc) [inline src/script/view_model/CallingViewModel.ts:233]: Isn't this logic also included in the `subscribeToEpochUpdates` call? I believe, when subscribed, it also get the current version, am I wrong?

**Review by atomrc — APPROVED** [review]
neat 😎

File: src/script/calling/mlsConference.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    members: SubconversationEpochInfoMember[];
+    epoch: number;
+    secretKey: string;
+    keyLength: number;
+  }) => void,
+): Promise<() => void> => {
+  const initialSubconversation = await mlsService.joinConferenceSubconversation(conversationId);
+
+  const forwardNewEpoch = async ({groupId, epoch}: {groupId: string; epoch: number}) => {
+    const subconversation = await mlsService.getConferenceSubconversation(conversationId);
```
Comment (atomrc) [inline src/script/calling/mlsConference.ts:82]: We should probably move this after the `if` condition, in order not to fetch the conversation if not needed

**Pull request merged**